src/types/testing.ts

```diff
@@ -42,17 +45,17 @@ export interface GetTestAccountParams {
   initialFunds: AlgoAmount
   /** Whether to suppress the log (which includes a mnemonic) or not (default: do not suppress the log) */
   suppressLog?: boolean
-  /** Optional override for how to get an account; this allows you to retrieve accounts from a known or cached list of accounts. */
-  accountGetter?: (algod: Algodv2, kmd?: Kmd) => Promise<Account>
+  /** Optional override for how to get a test account; this allows you to retrieve accounts from a known or cached list of accounts. */
+  accountGetter?: (algorand: AlgorandClient) => Promise<Account>
```

Comment:
Technically this is a breaking change, but it's a very recent feature that wasn't advertised and I'm pretty confident it's unlikely people will be using it.